app/controllers/grocer/exports_controller.rb

```diff
@@ -37,7 +38,7 @@ def set_export
     # Only allow a trusted parameter "white list" through.
     def export_params
       params.require(:export).permit(:pid, :job, :status, :last_error, :last_success, :logfile,
-                                     :page)
+                                     :page, :ark)
```

Comment:
Will ARKs work as the unique key (I assume that's what this is for)? If we plan to export before the object is finished in some cases (didn't we?) If we don't mint an ARK until the object is "finished"...